web/src/app/components/cdf/cdf.component.ts

```diff
@@ -143,25 +89,24 @@ export class CDFComponent implements AfterViewInit {
       .attr('width', elementWidth)
       .attr('height', elementHeight);
 
-    const cdfChart = this.svg
+    const CDFChart = this.svg
```

Comment:
Why capitalizing CDFChart? We should only do `CDFChart` when the first letter `c` needs to be capitalized.